index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio Website</title>
+</head>
+<body>
+  <header id="container">
+    <div class="head">
+        <h1 class="logo">Suleiman</h1>
+    </div>
+    <nav>
+        <img src="images/Hamburguer.png" alt="an image here" class="image">
+    </nav>
+</header>
+
+<section>
+    <h2>I'm Suleiman Welcome To My Portfolio Website</h2>
+
+     <p>I'm a software developer and i can help you build a solution, implement a feature or build 
```

Comment:
- [ ] kindly use the correct background image like in the figma design
- [ ] please respect the figma margins. kindly increase your margin right

yours
![Screenshot (631)_LI](https://user-images.githubusercontent.com/65298048/204801030-73aace95-deed-4dd7-b3d9-4ff60ebb32c1.jpg)
figma
![Screenshot (632)](https://user-images.githubusercontent.com/65298048/204801147-eceb1099-2931-41f8-af10-dc84ef840ec3.png)